kotlinx-coroutines-test/MIGRATION.md

```diff
@@ -0,0 +1,324 @@
+# Migration to the new kotlinx-coroutines-test API
+
+In version 1.6.0, the API of the test module changed significantly.
+This is a guide for gradually adapting the existing test code to the new API.
+This guide is written step-by-step; the idea is to separate the migration into several sets of small changes.
+
+## Remove custom `UncaughtExceptionCaptor`, `DelayController`, and `TestCoroutineScope` implementations
+
+We couldn't find any code that defined new implementations of these interfaces, so they are deprecated. It's likely that
+you don't need to do anything for this section.
```

Comment:
It means that either this section should be moved to the bottom or the link to the next relevant section should be added

Reply:
Not quite, these are listed step-by-step, and this step is the first one to perform.

Reply:
Maybe then it's worth adding a link to a next paragraph?

Reply:
Does this idea generalize to adding a link to the next section to every section? If not, I don't get it.

Reply:
Well, it's not; this section is just likely to be skipped by a large portion of readers, thus the suggestion. I do not insist though